src/Sulu/Bundle/AdminBundle/Resources/js/components/Chip/tests/Chip.test.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import React from 'react';
+import {render, shallow} from 'enzyme';
+import Chip from '../Chip';
+
+test('Should render item with children', () => {
+    expect(render(<Chip value={{}}>Name</Chip>)).toMatchSnapshot();
+});
+
+test('Should render item with delete icon', () => {
```

Comment:
should we add a test case to check if the item is rendered `clickable` if an `onClick` callback is giben?